util/logger/logger.go

```diff
@@ -65,38 +66,48 @@ func getLoggersInst() *logger {
 }
 
 func InitGlobalLogger(conf *Config) {
-	if globalInst == nil {
-		writers := []io.Writer{}
-		// console writer
+	if globalInst != nil {
+		return
+	}
+
+	writers := []io.Writer{}
+	// console writer
+	_, err := os.Stderr.Write([]byte{0})
```

Comment:
// Check if stderr is null. If true, create a buffer writer to redirect log output from the logger to the file writer,
// ensuring log file creation: https://github.com/pactus-project/pactus/issues/1077

Reply:
Use this comment instead